src/fplan/fplan.py

```diff
@@ -43,15 +43,18 @@ def load_file(self, file):
 
         # 2023 tax table (could predict it moves with inflation?)
         # married joint at the moment, can override in config file
-        self.taxrates = d.get('taxrates',
-                              [[0,      0.00],
-                               [0.1,    0.10], # fake level to fix 0
-                               [22000,  0.12],
-                               [89450 , 0.22],
-                               [190750, 0.24],
-                               [364200, 0.32],
-                               [462500, 0.35],
-                               [693750, 0.37]])
+        tmp_taxrates = d.get('taxrates',
+                              [[0,      10],
+                               [22000,  12],
+                               [89450 , 22],
+                               [190750, 24],
+                               [364200, 32],
+                               [462500, 35],
+                               [693750, 37]])
+        # add fake level and swith to decimals
+        tmp_taxrates[:0] = [[0, 0]]
```

Comment:
This is the line that adds the [0,0] level so we don't have to ask the users to put in a fake level.